Kickstarter-iOS/Views/Controllers/CreatePasswordViewController.swift

```diff
@@ -0,0 +1,115 @@
+import Library
+import Prelude
+import UIKit
+
+private struct ReuseIdentifier {
+  static let cell = "SettingsInputCell"
+  static let header = "SettingsHeaderView"
+  static let footer = "SettingsFooterView"
+}
+
+private enum CreatePasswordRow: CaseIterable {
+  case newPassword
+  case confirmNewPassword
+}
+
+extension CreatePasswordRow {
+  var title: String {
+    switch self {
+    case .newPassword: return Strings.New_password()
+    case .confirmNewPassword: return Strings.Confirm_password()
+    }
+  }
+
+  var placeholder: String {
+    return Strings.signup_input_fields_password()
+  }
+}
+
+class CreatePasswordViewController: UITableViewController {
+  // MARK: - Accessors
+
+  private lazy var rightBarButtonItem: UIBarButtonItem = {
+    return UIBarButtonItem(barButtonSystemItem: .save, target: nil, action: nil)
+      |> \.isEnabled .~ false
+  }()
+
+  // MARK: - Lifecycle
+
+  override func viewDidLoad() {
+    super.viewDidLoad()
+
+    _ = self
+      |> \.title %~ { _ in Strings.Create_password() }
+
+    _ = self.navigationItem
+      |> \.rightBarButtonItem .~ self.rightBarButtonItem
+
+    _ = self.tableView
+      |> \.allowsSelection .~ false
+      |> \.separatorInset .~ .zero
+
+    if #available(iOS 11, *) { } else {
```

Comment:
Unfortunately necessary for iOS 10. Otherwise it doesn't know how to size the header, the cell nor the footer.